src/resources/platform-adaptations.md

```diff
@@ -560,6 +565,180 @@ This section includes preliminary recommendations on how to adapt
 Material widgets to deliver a natural and compelling experience on iOS. 
 Your feedback is welcomed on [issue #8427][]. 
 
+### Widgets with .adaptive() constructors
+
+Several widgets support `.adaptive()` constructors. 
+Adaptive constructors substitute the corresponding Cupertino components 
+when the app is run on an iOS device. 
+
+Widgets in the following table are used primarily for input, 
+selection, and to display system information. 
+Because these controls are tightly integrated with the operating system,
+users have been trained to recognize and respond to them.
+Therefore, we recommend that you follow platform conventions. 
+
+
+| Material Widget | Cupertino Widget | Adaptive Constructor |
+|---|---|---|---|---|
+|<img width=160 src="/assets/images/docs/platform-adaptations/m3-switch.png" class="figure-img img-fluid rounded" alt="Switch in Material 3" /><br/>`Switch`|<img src="/assets/images/docs/platform-adaptations/hig-switch.png" class="figure-img img-fluid rounded" alt="Switch in HIG" /><br/>`CupertinoSwitch`|[`Switch.adaptive()`][]|
+|<img src="/assets/images/docs/platform-adaptations/m3-slider.png" width =160 class="figure-img img-fluid rounded" alt="Slider in Material 3" /><br/>`Slider`|<img src="/assets/images/docs/platform-adaptations/hig-slider.png"  width =160  class="figure-img img-fluid rounded" alt="Slider in HIG" /><br/>`CupertinoSlider`|[`Slider.adaptive()`][]|
+|<img src="/assets/images/docs/platform-adaptations/m3-progress.png" width = 100 class="figure-img img-fluid rounded" alt="Circular progress indicator in Material 3" /><br/>`CircularProgressIndicator`|<img src="/assets/images/docs/platform-adaptations/hig-progress.png" class="figure-img img-fluid rounded" alt="Activity indicator in HIG" /><br/>`CupertinoActivityIndicator`|[`CircularProgressIndicator.adaptive()`][]|
+| <img src="/assets/images/docs/platform-adaptations/m3-checkbox.png" class="figure-img img-fluid rounded" alt=" Checkbox in Material 3" /> <br/>`Checkbox`| <img src="/assets/images/docs/platform-adaptations/hig-checkbox.png" class="figure-img img-fluid rounded" alt="Checkbox in HIG" /> <br/> `CupertinoCheckbox`|[`Checkbox.adaptive()`][]|
+|<img src="/assets/images/docs/platform-adaptations/m3-radio.png" class="figure-img img-fluid rounded" alt="Radio in Material 3" /> <br/>`Radio`|<img src="/assets/images/docs/platform-adaptations/hig-radio.png" class="figure-img img-fluid rounded" alt="Radio in HIG" /><br/>`CupertinoRadio`|[`Radio.adaptive()`][]|
+
+### Top app bar and navigation bar
+
+Since Android 12, the default UI for top app 
+bars follow the design guidelines defined in [Material 3][mat-appbar]. 
+On iOS, an equivalent component called "Navigation Bars" 
```

Comment:
Please put the link on "Navigation Bars". But keep the "Apple's Human..." text. Just don't link on that text.